### ISSUE TITLE
Question about Products showing up inside the Sanity Studio 

### ISSUE THREAD

**Issue by petercr** [ndimatteo/HULL#3]
### Great project 🤘 Quick question about the Products syncing in Sanity

First off, **great job** on this project 💯 🔥 🔥
I have been looking through the code, and really digging it.

So I have the whole project up and running locally with the Sanity Studio server running on `http://localhost:3333` and I have the Next site running locally on `http://localhost:3000`.

When I go to the `Shop` section of the Studio, everything looks to be working fine. 
I can see listings of `Products` there, and I can see listings under `Product Variants`.
But _only products_ that have variants are getting listed in either section. 
If I add a product on Shopify **_without_** variants, then I don't see it in the Studio. 
It looks like it is just doubling up the `Product Variant` data instead of having 2 different types of product data.

I didn't know if you had any idea as to why this was happening, or how best to debug it?
Any insight would be appreciated 👍 

Thanks 😄 

Here are some screenshots showing I get the views and url's for both Products and Product Variants.

Product View:
![sanity-products](https://user-images.githubusercontent.com/26460352/106821303-e2959900-664a-11eb-9be8-ff89cd2a7cba.png)

Product Variant View:
![sanity-product-variant](https://user-images.githubusercontent.com/26460352/106821325-ee815b00-664a-11eb-94e9-d48fea3a974e.png)

**Comment by ndimatteo** [comment]
Hey there @petercr thanks for the kind comments! 🙏 I'm glad you're digging using HULL!! 🤘 

Your screenshots look to be displaying product data properly, but in your screenshot of the "Product Variants" view, you have to click into a product to see the associated variants for it (in case that wasn't clear). If there are no variants set in Shopify for a given product, there will still technically be _one_ variant (the only variant) for the product (this is even how Shopify denotes products in their data structure).

All products, regardless of variants will have a base product document in Sanity, which are in the "Product" view, and is what the "Product Variants" are grouped by.

Also, remember that products are sync'd using the shopify webhooks which hit the shopify sync api endpoint for Next (`/api/shopify/product-update`). So if you had products created before the webhooks and sync were setup, those products will not get pulled into Sanity until you make some kind of change to them on Shopify.

If that all makes sense, maybe I'm still misunderstanding what you're experiencing, so please let me know!

Perhaps it would help to know what you're expecting to see, vs what you're actually seeing?

Let me know if that helps, looking forward to hearing more!

**Comment by petercr** [comment]
@ndimatteo Thanks so much for getting back to me so quickly 🚀 

I feel like _SUCH_ a noob 😆 so you were right about this part:
>Also, remember that products are sync'd using the shopify webhooks which hit the shopify sync api endpoint for Next (/api/shopify/product-update). So if you had products created before the webhooks and sync were setup, those products will not get pulled into Sanity until you make some kind of change to them on Shopify.

I had a product that was in Shopify and was not showing up in Sanity. 
That is was I was trying to get across with the previous message. 
I call it "The Case of The Missing Pants" 🔍 

I went to the Shopify Admin section, and updated some of the info on the product. (price & inventory) 
Then I went back to Sanity and check the products again, and there it was 🤦 
It seems it was just a matter of needing to update a value on the product, and the web hook picked it up 🔥 

Thanks again for the help, the issue is all set now 😎
